data/names/zh_CN.json

```diff
@@ -1,136 +1,1827 @@
+{"usage": "nick", "name": "维加斯"},
+{"usage": "nick", "name": "报复"},
+{"usage": "nick", "name": "威尼斯"},
+{"usage": "nick", "name": "毒液"},
+{"usage": "nick", "name": "超大杯"},
+{"usage": "nick", "name": "金星"},
+{"usage": "nick", "name": "眩晕"},
+{"usage": "nick", "name": "才能"},
+{"usage": "nick", "name": "否决"},
+{"usage": "nick", "name": "烦恼"},
+{"usage": "nick", "name": "胜利"},
+{"usage": "nick", "name": "取景器"},
+{"usage": "nick", "name": "维京人"},
+{"usage": "nick", "name": "醋"},
+{"usage": "nick", "name": "贵宾"},
+{"usage": "nick", "name": "毒蛇"},
+{"usage": "nick", "name": "伏特"},
+{"usage": "nick", "name": "志愿者"},
+{"usage": "nick", "name": "伏都教"},
+{"usage": "nick", "name": "沃克斯"},
+{"usage": "nick", "name": "秃鹰"},
+{"usage": "nick", "name": "瓦克"},
+{"usage": "nick", "name": "华夫饼"},
+{"usage": "nick", "name": "醒来"},
+{"usage": "nick", "name": "沃克"},
+{"usage": "nick", "name": "壁花"},
+{"usage": "nick", "name": "减弱"},
+{"usage": "nick", "name": "水性杨花的女人"},
+{"usage": "nick", "name": "战争"},
+{"usage": "nick", "name": "监狱长"},
+{"usage": "nick", "name": "军阀"},
+{"usage": "nick", "name": "征途"},
+{"usage": "nick", "name": "疣猪"},
+{"usage": "nick", "name": "黄鼠狼"},
+{"usage": "nick", "name": "楔子"},
+{"usage": "nick", "name": "周三"},
+{"usage": "nick", "name": "怪人"},
+{"usage": "nick", "name": "西德佬"},
+{"usage": "nick", "name": "西"},
+{"usage": "nick", "name": "西部人"},
+{"usage": "nick", "name": "西部佬"},
+{"usage": "nick", "name": "老生常谈的"},
+{"usage": "nick", "name": "反复无常"},
+{"usage": "nick", "name": "旋风"},
+{"usage": "nick", "name": "威士忌"},
+{"usage": "nick", "name": "耳语"},
+{"usage": "nick", "name": "白色"},
+{"usage": "nick", "name": "奇才"},
+{"usage": "nick", "name": "威兹班"},
+{"usage": "nick", "name": "哇"},
+{"usage": "nick", "name": "为什么"},
+{"usage": "nick", "name": "小部件"},
+{"usage": "nick", "name": "摆动"},
+{"usage": "nick", "name": "野生"},
+{"usage": "nick", "name": "莽撞的人"},
+{"usage": "nick", "name": "懦弱的"},
+{"usage": "nick", "name": "饶舌之人"},
+{"usage": "nick", "name": "翅膀"},
+{"usage": "nick", "name": "剔出"},
+{"usage": "nick", "name": "惨败"},
+{"usage": "nick", "name": "线"},
+{"usage": "nick", "name": "电线"},
+{"usage": "nick", "name": "自以为聪明者"},
+{"usage": "nick", "name": "向导"},
+{"usage": "nick", "name": "狼"},
+{"usage": "nick", "name": "狼人"},
+{"usage": "nick", "name": "想知道"},
+{"usage": "nick", "name": "书呆子"},
+{"usage": "nick", "name": "旺卡"},
+{"usage": "nick", "name": "靠不住的"},
+{"usage": "nick", "name": "汪"},
+{"usage": "nick", "name": "兰开夏人"},
+{"usage": "nick", "name": "老王"},
+{"usage": "nick", "name": "词"},
+{"usage": "nick", "name": "蠕虫"},
+{"usage": "nick", "name": "哇"},
+{"usage": "nick", "name": "幽灵"},
+{"usage": "nick", "name": "忿怒"},
+{"usage": "nick", "name": "造成"},
+{"usage": "nick", "name": "沉船"},
+{"usage": "nick", "name": "肇事者"},
+{"usage": "nick", "name": "坏蛋"},
+{"usage": "nick", "name": "韩国帝王"},
+{"usage": "nick", "name": "Xi"},
+{"usage": "nick", "name": "x射线"},
+{"usage": "nick", "name": "Yadda Yadda"},
```

Comment:
👀

Reply:
Yeah, that one is also in the en.json, I choose not to change it.